addon/components/frost-buckets.js

```diff
@@ -6,7 +6,7 @@ import layout from '../templates/components/frost-buckets'
 import PropTypesMixin, {PropTypes} from 'ember-prop-types'
 const {createStore} = redux
 import _ from 'lodash'
-import redux from 'npm:redux'
```

Comment:
Lets leave this alone and keep `"ember-browserify": "1.1.13",` installed as well for now.